lib/config.js

```diff
@@ -29,15 +36,16 @@ if (argv.config) {
   if (typeof argv.config === 'string') {
     argv.config = [argv.config];
   }
-  for (var i in argv.config) {
-    loader.add(path.resolve(argv.config[i]));
-  }
+
+  argv.config.forEach(function(val) {
+    loader.add(path.resolve(val));
+  });
 }
 
 var setOptions = {};
 var key = null;
 for (key in yargs.argv) {
-  if (yargs.argv[key] !== undefined) {
+  if (!_.isUndefined(yargs.argv[key])) {
```

Comment:
I actually feel almost the total opposite. Lodash has a bunch of good features that prevent problems that using native JS tends to introduce. I think it is always a bad Idea to be looping over keys in this manner anyway and would was simply trying to get this to pass our linting setup without rewriting it.